### PR TITLE
Disabled the bus tests and added json error message

### PR DESCRIPTION
**Pull request by kristjanmik**

### CODE REVIEW

**Comment by kristjanmik** [comment]
It looks like the bus data source fixed itself. I'm adding the tests back. (This)[https://github.com/apis-is/apis/commit/0400634e0b0e0dbb26c9e9dbe6a65e6dd990a553#diff-30d7399c2b5371e04d892af99233d0bbR20] needs to be merged to catch the error in the future

**Comment by MiniGod** [comment]
The bus endpoint have caused the nightly builds to go red and green at random the last couple of days. It's not stable.
IMO it's ok to disable for now. Use `describe.skip` instead of commenting it out.

**Comment by kristjanmik** [comment]
Ok, skipping for now...

**Comment by koddsson** [comment]
This LGTM, can it be merged?

File: endpoints/bus/realtime.js [hunk 1]
```diff
@@ -17,6 +17,13 @@ app.get('/bus/realtime', function(req, res){
 			return res.json(500,{error:'Something is wrong with the data provided from the data source'});
 		}
 
+    if(obj.error){
+      var hasDescription = obj.error && obj.error.desc;
+
+      return res.status(500).json({
+        error: hasDescription ? obj.error.desc : 'Something went wrong'
+      });
+    }
```
Comment (MiniGod) [inline endpoints/bus/realtime.js]: Different indentation?

**Pull request closed without merging**